spec/asyncapi.yaml

```
@@ -0,0 +1,14 @@
+info:
+  title: myapp events
+  version: 0.1.0
+channels:
+  ws/users:
+    address: /ws
+    messages:
+      UserCreated:
+        payload:
+          type: object
+          properties:
+            id: { type: string }
```

YAMLlint (1.37.1)

[error] too many spaces inside braces

(braces)